turbinia/config/rules/suspicious_cron.yar

```diff
@@ -0,0 +1,18 @@
+rule suspicious_cron {
+    meta:
+        description = "Rule to find persistance in Cron files"
+        author = "Fry"
+        date = "2022-07-01"
+        score = 90
+    strings:
+        $wget = "wget" nocase
+        $options1 = "-q"
+        $options2 = "-O-"
+        $curl1 = "curl" nocase
+        $curl2 = "-fssl" nocase
+        $shm = "/dev/shm"
+        $pipechar = "|"
+        $shell = "sh"
+   condition:
+      (filepath matches /cron/ or filepath matches /spool\/at/) and ($wget or any of ($options*) or any of ($curl*) or $shm or ($pipechar and $shell))
```

Comment:
I'm not very familiar with the syntax, but the `($wget or any of ($options*) or` section of this reads to me like this would trigger if `-q` was anywhere in any of the cron/at files.  Seems like that could be FP prone.  Or am I reading that wrong?

Reply:
Great point & good catch - I had a case where wget was renamed (to "w" or something) and the only thing that made it look like wget was the `-q` flag..but I agree that would be too prone.
I've modified the rule so that it needs both options if there's no wget string, which should cut down on FPs.
I've also changed it so that the shell has to appear after the pipe, which in its current state matches on normal out of the box cron files.